source/connect/csot.txt

```diff
@@ -0,0 +1,234 @@
+.. _ruby-csot:
+
+===========================
+Limit Server Execution Time
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: error, blocking, thread, task
```

Comment:
S: always include code example in this list if there are code examples on the page!